db/structure.sql

```diff
@@ -844,6 +858,13 @@ CREATE INDEX index_measurement_imports_on_subtype ON measurement_imports USING b
 CREATE INDEX index_measurements_on_captured_at ON measurements USING btree (captured_at);
 
 
+--
+-- Name: index_measurements_on_captured_at_and_unit_and_device_id; Type: INDEX; Schema: public; Owner: -
+--
+
+CREATE INDEX index_measurements_on_captured_at_and_unit_and_device_id ON measurements USING btree (captured_at, unit, device_id) WHERE (device_id IS NOT NULL);
```

Comment:
Seems like we're making perf changes along with a rails 4.2 upgrade here. Are we sure we want to do this?

I could be reading this wrong but pretty sure each of these sort of indices are ~3GB

```
                            relname                             | relpages
----------------------------------------------------------------+----------
 idx_measurements_captured_at_unit_device_id_device_id_not_null |   172609
 idx_measurements_value_device_id_device_id_not_null            |   155725
 index_measurements_on_captured_at                              |   325448
 index_measurements_on_device_id                                |   318700
 index_measurements_on_location                                 |  1166141
 index_measurements_on_md5sum                                   |   829269
 index_measurements_on_measurement_import_id                    |   341421
 index_measurements_on_original_id                              |   380629
 index_measurements_on_unit                                     |   336601
 index_measurements_on_user_id                                  |   331424
 index_measurements_on_user_id_and_captured_at                  |   472871
 index_measurements_on_value_and_unit                           |   430323
 measurements_pkey                                              |   250499
(13 rows)
```

Reply:
As discussed in #425, these indexes are indexes without migration. They must be added by hand. Originally, I created new migration file. But, it would cause to create new indexes when deploying due to names of those indexes are different. So, I put statements to create indexes in last migration.